samples/gradle-plugin/build.gradle.kts

```diff
@@ -0,0 +1,14 @@
+tasks {
+
+    val publishPlugin by creating(GradleBuild::class) {
+        dir = file("plugin")
+        tasks = listOf("publish")
+    }
+
+    val checkSample by creating(GradleBuild::class) {
```

Comment:
:x: The task in the sample should make sense on its own. I think `consumePlugin` would be a better name, see below.